README.md

```diff
@@ -42,11 +42,15 @@ Use a recent version of [Apache Maven](http://maven.apache.org/download.cgi) (e.
     cd utils-java
     mvn package
 
-If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java), just
+If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java)), just
 
   1. Build and install the jar to your local maven repository via `mvn install`.
   2. Then update the pom.xml of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.
-
+  3. Eclipse users: 
+    * First update the dependencies in your poms to point to the SNAPSHOT versions as mentioned above.
```

Comment:
Isn't (3.a) redundant with point (2)? I think we can just remove it. Other than that, the indentation totally cleared up the confusion, thank you!